__tests__/app.test.js

```diff
@@ -65,6 +65,7 @@ describe("GET", () => {
           expect(article[0]).toHaveProperty("created_at");
           expect(article[0]).toHaveProperty("votes");
           expect(article[0]).toHaveProperty("article_img_url");
+          expect(article[0]).toHaveProperty("comment_count")
```

Comment:
This should be a new test - it is new behaviour. Updating tests for new behaviour is not TDD. If you _had_ to update this test then it means the test was "brittle". Ideally it should have been originally written in a way that asserted correctly but didn't fail for added behaviours.